scripts/ci/run-integration-tests

```diff
@@ -43,8 +43,17 @@ function runTestFile {
   exit 1
 }
 
-echo 'Running tests per file (./tests/*-cy.js)'
-find $PROJECT_ROOT/tests -type f -name '*-cy.js' | while read file; do runTestFile "$file"; done
+
+# Starting http-server & setting up trap
+http-server -sp 4200 dist&
+SERVER_PID=$!
+trap "kill $SERVER_PID" EXIT
+
+# pass in simple search strings for filenames here (find syntax)
+SEARCH=${1:-'*'}
+
+echo "Found these matching files ('./tests/${SEARCH}-cy.js')"
+find $PROJECT_ROOT/tests -type f -name ${SEARCH}"-cy.js" -print | tee /dev/tty | while read file; do runTestFile "$file"; done
```

Comment:
What does `tee /dev/tty` do?

Reply:
its a hack to be able to pipe (stdout, one output of `tee`) AND have it on the current terminal (which is - file wise - `/dev/tty`, the file `tee` writes to)

Reply:
Thank you for explaining :)